src/Sepes.RestApi/web.config

```diff
@@ -1,13 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-
+  
   <system.webServer>
     <security>
       <requestFiltering>
         <!-- Handle requests up to 1 GB -->
-        <requestLimits maxAllowedContentLength="1073741824" />
+        <requestLimits maxAllowedContentLength="2147483648" />
       </requestFiltering>
     </security>
   </system.webServer>
+  <system.web>
+    <httpRuntime targetFramework="4.7.2" executionTimeout="9999" maxRequestLength="2000000" />
```

Comment:
This targetFramework=... looks like it belongs in a .NET Framework application and should probably be removed, but unsure if the executonTimeout and maxRequestLength can still be there